docs/source/email_setup.rst

```diff
@@ -0,0 +1,115 @@
+Email Notification Setup
+========================
+
+Dallinger can be configured to send email messages when errors occur during a
+running experiment. If this configuration is skipped, messages which
+would otherwise be emailed will be written to the experiment logs instead.
+
+Instructions
+-------------
+Sending email from Dallinger requires 5 configuration settings, described in
+turn below. Like all configuration settings, they can be set up in either
+`.dallingerconfig` in your home directory, or in `config.txt` in the root
+directory of your experiment.
+
+The Config Settings
+~~~~~~~~~~~~~~~~~~~
+
+``smtp_host`` The hostname and port of the SMTP (outgoing email) server through
+which all email will be sent. This defaults to `smtp.gmail.com:587`, the Google
+SMTP server. If you want to send email *from* a Gmail address, or a custom
+domain set up to use Gmail for email, this default setting is what you want.
+
+``smtp_username`` The username with which to log into the SMPT server, which
+will very likely be an email address (if you are using a Gmail address to send
+email, you will use that address for this value).
+
+``smpt_password`` The password associated with the ``smtp_username``. **NOTE**
+If you are using two-factor authentication, see :ref:`two-factor-auth`, below.
+
+``dallinger_email_address`` The email address to be used as the "from" address
+outgoing email notifications. For Gmail accounts, this address is likely to be
+overwritten by the Google SMTP server. See :ref:`from-address-rewrite` below.
+
+``contact_email_on_error`` Also an email address, and used in two ways:
+
+1. It serves as the recipient address for outgoing notifications
+2. It is displayed to experiment participants on the error page, so that
+   they can make inquiries about compensation
+
+
+Pitfalls and Solutions
+~~~~~~~~~~~~~~~~~~~~~~
+
+A few other things which may get in the way of sending email successfully, or
+cause things to behave differently than expected:
+
+
+.. _two-factor-auth:
+
+Two-Factor Authentication
+"""""""""""""""""""""""""
+
+Having two-factor authentication enabled for the outgoing email account will
+prevent dallinger from sending email without some additional steps. Details
+instructions are provided for Gmail, below. Other email services which support
+two-factor authentication may provide equivalent solutions for marking an app
+as safe to send email.
+
```

Comment:
The grammar is a bit funny (to me) here?
Other email services which support two-factor authentication may provide equivalent solutions for marking an app
as safe to send email.

Reply:
I agree. It became tortured in part because I was trying to avoid the problematic phrase "whitelist". Will rework.

Reply:
Just going to end the sentence at "... solutions."